BUILDING.md

```diff
@@ -30,13 +30,32 @@ To build with only the TinkerPop tests:
 mvn clean install -Dtest.skip.tp=false -DskipTests=true
 ```
 
+To build the distribution archive:
+
+```
+mvn clean install -Pjanusgraph-release -Dgpg.skip=true -DskipTests=true
+```
+This command generates the distribution archive in `janusgraph-dist/janusgraph-dist-hadoop-2/target/janusgraph-$VERSION-hadoop2.zip`.
+For more details information, please see [here](janusgraph-dist/README.md#building-zip-archives)
+
 ## Building Docker Image for JanusGraph Gremlin Server
 
-To build and run Docker images with JanusGraph and Gremlin Server, configured 
+In order to build Docker image for JanusGraph Gremlin Server, a
```

Comment:
Please change this to _JanusGraph Server_ to avoid confusion as that's what we're using in the docs.

Reply:
I pulled this section from master, and I agree it sounds better as JanusGraph Server. I updated line 33 in addition to the 2 you pointed out. Thanks for the review.